### PR TITLE
Revert "Add changeRequestEnabled to project and project_environments"

### PR DESCRIPTION
**Pull request by andreas-unleash**
Reverts Unleash/unleash#2357

### CODE REVIEW

**Review by kwasniew — APPROVED** [review]
I'm not sure if we can delete existing migrations or we should write another one. I remember we had a discussion to make migrations an immutable thing. But since that's the latest one maybe we can do it.

File: src/migrations/20221107132528-change-request-project-options.js [hunk 1]
```diff
@@ -1,21 +0,0 @@
-'use strict';
```
Comment (ivarconr) [inline src/migrations/20221107132528-change-request-project-options.js:1]: ⚠️ **DANGER:**  We need to be careful removing migrations.

Comment (ivarconr) [reply src/migrations/20221107132528-change-request-project-options.js:1]: "okish" as long as we have done 0 releases on it.

**Comment by andreas-unleash** [comment]
Reverting was done by creating a new migration to fix - This is unnecessary

**Pull request closed without merging**